src/database/schema/ChatSchema.ts

```diff
@@ -31,6 +31,8 @@ class ChatSchema implements BaseDbSchema {
 
   isContinuous: boolean;
 
+  attachmentJson: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` field is declared as `any`. It's a good practice to avoid using `any` type in TypeScript. If possible, define an interface for the structure of the attachment JSON.

```diff
-  attachmentJson: any;
+  attachmentJson: AttachmentInterface; // Define this interface according to your needs
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->